### ISSUE TITLE
Get Started Quickly Templates: CSS Too Complicated

### ISSUE THREAD

**Issue by rhroyston** [ampproject/ampstart#799]
ISSUE:
3,500 lines of CSS (when formatted for IDE viewing/development) in the [Gallery Quick Start Template](https://www.ampstart.com/archive/gallery.zip) is too much. 3,800 lines of CSS in the E-Commerce template.

RECOMMENDED SOLUTION:
Provide a simple getting started template to make AMP-HTML more accessible.

![image](https://user-images.githubusercontent.com/12751611/35594482-f69fb956-05d8-11e8-8591-c2324229836f.png)

![image](https://user-images.githubusercontent.com/12751611/35596346-20217afc-05df-11e8-91b7-e57aaf127ccf.png)

**Comment by cathyxz** [comment]
@rhroyston Could you provide more context on what would a good "getting started" template look like for your use case? For example, for basic styling of commonly used components, there's also: https://ampstart.com/components, for just a bare bones boilerplate amp, there's this [codelab](https://www.ampproject.org/docs/tutorials/create/basic_markup). ampbyexample.com should have basic (no styling) usage examples for each component. Is there something in between that would work better for your needs?

**Comment by rhroyston** [comment]
I reviewed the info you shared. Thank you. re https://ampstart.com/components, I initially don't see where to copy/paste from but did figure out I needed to click the "HTML" chevron to get the actual HTML code. Maybe auto expand that first one to highlight that the HTML is hidden until expanded? (or maybe it's just me!). Anyways, thanks for the response. I simply found the getting started templates (gallery, e-commerce) overwhelming and left it alltogether. I'll circle back on my next project though. Thanks again.
